README.md

```diff
@@ -0,0 +1,31 @@
+# Developer Tooling
+
+> **NOTE** This is a relatively new repository and is still under active development.
+> Some of the packages and release process may not be fully documented yet.
```

Comment:
"release process" => release proces**ses**" 

to keep symmetry

Reply:
Fixed in d646f8003a25fd533c17822cd0180f502a1e79cc ✅